tests/lib/rules/require-meta-type.ts

```diff
@@ -66,16 +66,6 @@ ruleTester.run('require-meta-type', rule, {
         create(context) {}
       };
     `,
-    {
-      code: `
-        const create = {};
-        module.exports = {
-          meta: {},
-          create,
-        };
-      `,
-      errors: [{ messageId: 'missing' }],
-    },
```

Comment:
What was this removed for?

Reply:
This was in the `valid` array, but had an `errors` property, which isn't correct.  Should I add it to the invalid array?

Reply:
I'm not sure if it's valid or invalid, but whichever one it is, it would be good to put it there.

Reply:
Cool.  I'll add it to the `invalid` array.

Reply:
Oh, actually, nevermind.  Its `create` doesn't have a `context`, so it's not detecting it as a rule.  I.e. `valid`.  I'll add it back to valid and remove the `errors`.

Reply:
Done